doc: add minutes for meeting 16 April (#850)
nodejs/TSC: meetings/2020-04-16.md

* Anna Henningsen @addaleax (TSC)
* Beth Griggs @BethGriggs (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Gabriel Schulhof @gabrielschulhof (TSC)
* Matteo Collina @mcollina (TSC)
* Matheus Marchini @mmarchini (TSC)
* Michael Dawson @mhdawson (TSC)
* Michaël Zasso @targos (TSC)
* Myles Borins @MylesBorins (TSC)
* Shelley Vohr @codebytere (TSC)
* Rich Trott @Trott (TSC)
* Ruben Bridgewater @BridgeAR (TSC)
* Tobias Nießen @tniessen (TSC)

## Agenda

### Announcements

### CPC and Board Meeting Updates
 
*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* Nominating @puzpuzpuz as a collaborator [#32555](https://github.com/nodejs/node/issues/32555)
  * Anna, onboarded earlier this week

* Nominating @juanarbol as a collaborator
[#32621](https://github.com/nodejs/node/issues/32621)  
  * Will be onboarded later this week.

### nodejs/docker-node

* Suggestion for dropping Yarn from Node 14 release [#1238](https://github.com/nodejs/docker-node/issues/1238)
  * Rich: Robust discussion in the issue, several TSC members have chimed in.

>     suggest continue discussion in the github issue, leave agenda tag on

Reviewer: ```suggestion
    Suggest continue discussion in the GitHub issue and leave the `tsc-agenda` tag on.
```

>     Suggest continue discussion in the GitHub issue and leave the `tsc-agenda` tag on.

Reviewer: ```suggestion
    Suggest continue discussion in the GitHub issue and leave the `tsc-agenda` tag on
```

>     Issue in case an impasse is reach, but does not look like we are there yet.

Reviewer: ```suggestion
    issue in case an impasse is reached, but it does not look like we are there yet.
```
